core/src/services/tikv/backend.rs

```diff
@@ -20,22 +20,25 @@ use std::fmt::Debug;
 use std::fmt::Formatter;
 
 use async_trait::async_trait;
+use serde::Deserialize;
 use tikv_client::Config;
 use tikv_client::RawClient;
 use tokio::sync::OnceCell;
 
 use crate::raw::adapters::kv;
+use crate::raw::*;
 use crate::Builder;
 use crate::Capability;
 use crate::Error;
 use crate::ErrorKind;
 use crate::Scheme;
 use crate::*;
 
-/// TiKV backend builder
-#[doc = include_str!("docs.md")]
-#[derive(Clone, Default)]
-pub struct TikvBuilder {
+/// Config for Mysql services support.
+#[derive(Default, Deserialize, Clone)]
+#[serde(default)]
+#[non_exhaustive]
+pub struct TikvConfig {
```

Comment:
Please make all fields `pub`.

Reply:
#[derive(Default, Deserialize, Clone)]
#[serde(default)]
#[non_exhaustive]
pub struct TikvConfig {
    /// network address of the TiKV service.
    pub endpoints: Option<Vec<String>>,
    /// whether using insecure connection to TiKV
    pub insecure: bool,
    /// certificate authority file path
    pub ca_path: Option<String>,
    /// cert path
    pub cert_path: Option<String>,
    /// key path
    pub key_path: Option<String>,
} 
this?